guidelines/sc/20/reading-level.html

```diff
@@ -4,7 +4,7 @@ <h4>Reading Level</h4>
    
    <p class="conformance-level">AAA</p>
    
-   <p>When text requires reading ability more advanced than the <a>lower secondary education level</a> after removal of proper names and titles, <a>supplemental content</a>, or a version that does not require reading ability more advanced than the lower
+  <p>When text requires reading ability more advanced than the <a>lower secondary education level</a> after removal of proper names and titles, <a>supplemental content</a>, or a version that does not require reading ability more advanced than the lower
```

Comment:
It is not clear why this paragraph is shown as a change.

Reply:
There is only a whitespace change. This was part of #3038

Reply:
Now I see it!  Space before `p`